recipes/sundials/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cc -I${PREFIX}/include -L${PREFIX}/lib -o cvRoberts_dns cvRoberts_dns.c -lm -lsundials_cvode -lopenblas -lsundials_nvecserial
```

Comment:
No problem. That is fine. I was just curious why you chose this path.

Maybe it is worth adding a comment on this file about this test and the reasons you chose to do so.